From: "Barak A. Pearlmutter" <barak+git@pearlmutter.net>
Date: Wed, 26 Aug 2015 13:28:57 +0100
Subject: AM_PROG_AR

Squelch automake warning about how AM_PROG_AR is needed for non-POSIX linkers
---
 autocrap/configure.ac | 1 +
 1 file changed, 1 insertion(+)

diff --git a/autocrap/configure.ac b/autocrap/configure.ac
index faebd70..575a20f 100644
--- a/autocrap/configure.ac
+++ b/autocrap/configure.ac
@@ -3,6 +3,7 @@ AC_CONFIG_AUX_DIR([config.aux])
 AM_INIT_AUTOMAKE([foreign subdir-objects -Wall])
 AC_PROG_CC
 AC_PROG_RANLIB
+AM_PROG_AR
 
 
 # Check for clock_gettime.  On some systems, this is provided via librt.
